src/components/DelegatesTable/DelegateVoteCheckbox.vue

```diff
@@ -93,13 +93,14 @@ export default {
   .delegate-vote-checkbox {
     &__icon {
       color: map-get($adm-colors, 'muted');
-    }
-  }
-}
 
-.dark {
-  .delegate-vote-checkbox {
-    &__icon {
+      &--good {
+        color: map-get($adm-colors, 'good');
+      }
+
+      &--danger {
+        color: map-get($adm-colors, 'danger');
+      }
```

Comment:
Thats correct, theme specific styles like `color` and `background-color` should be inside `.v-theme--light` and `.v-theme--dark` selectors.

1. So, you can remove `&--good` and `&--danger` classes from common styles (line 82-88)
2. Check if default colors in dark theme are ok. If not, define them in `.v-theme--dark`

Reply:
Done.